sirbot-pyslackers/endpoints/slack/commands.py

```diff
@@ -9,16 +9,32 @@
 
 def create_endpoints(plugin):
     plugin.on_command("/admin", tell_admin)
-    plugin.on_command("/howtoask", ask)
+    plugin.on_command("/howtoask", how_to_ask)
     plugin.on_command("/gif", gif_search)
+    plugin.on_command("/justask", just_ask)
     plugin.on_command("/pypi", pypi_search)
     plugin.on_command("/sponsors", sponsors)
     plugin.on_command("/snippet", snippet)
     plugin.on_command("/report", report)
     plugin.on_command("/save", save_conversation)
 
 
-async def ask(command, app):
+async def just_ask(command, app):
+    slack = app.plugins["slack"]
+    response = Message()
+    response["channel"] = command["channel_id"]
+    response["unfurl_links"] = False
+
+    response["text"] = (
+        "If you have a question, please just ask it. Do not ask to ask; do not ask for topic experts; "
+        "do not DM or ping random users. We cannot begin to answer a question until we actually get a "
+        "question. Please skip the formalities and just ask."
+    )
```

Comment:
Seems good to me but might be worth asking in #community and/or #admin for feedback

Reply:
already did in #community_projects, but will do in community as well.

Reply:
Changed this to Jabob's suggestion